GOVERNANCE.md

```diff
@@ -34,9 +34,10 @@ A PR should be created with all relevant changes to update the repository for th
 (See [#81](https://github.com/hamstar/Wikimate/pull/81) for an example.)
 It should apply the following actions:
 
-1. Change the "Unreleased version" heading in the CHANGELOG.md file
+1. Change the "Upcoming version" heading in the CHANGELOG.md file
    to the appropriate version name (e.g. "Version 1.2.3")
-   and add a new "Unreleased version" section heading above it;
+   and add a new "Upcoming version" section heading above it,
+   with the contents "No changes yet.";
```

Comment:
A belated and very tiny nitpick, but changelog entries normally don't end with a period, so this dummy entry shouldn't either.

Reply:
I can see where you're coming from, but on the other hand the changelog entries are formatted as bullet list items (which commonly don't use punctuation) while this is a regular sentence. I don't think it would make sense to format it as a list item, btw.

If you agree, I'll go ahead and add the period in #102.

Reply:
Ah, indeed - agreed on both, uhm, points. :smile:

Reply:
Great, let's resolve this thread then! I just pushed an update to #102.